test/test_pallas.py

```diff
@@ -941,46 +935,90 @@ def quantized_matmul_int8_wrapper(x, w_int, scalar, quantize_activation,
         w_int_xla,
         scalar_xla,
         quantize_activation=quantize_activation,
-        batch_block_size=batch_block_size,
-        out_block_size=out_block_size,
-        in_block_size=in_block_size).cpu()
+    ).cpu()
+
+    print(f'Output max diff: {torch.max(torch.abs(expected - actual))}')
```

Comment:
Do we need to delete these prints?

Reply:
I found it very helpful. Let me comment it out and leave it there.